data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -153,11 +147,34 @@ $IPTABLES -t raw -A DHCP_IRONIC -j DROP
 {{end}}
 
 # Wait for images to be downloaded/ready
-podman wait -i 1000ms ipa-downloader
+#podman wait -i 1000ms ipa-downloader
 podman wait -i 1000ms coreos-downloader
-while ! curl --fail http://localhost/images/rhcos-ootpa-latest.qcow2.md5sum ; do sleep 1; done
-while ! curl --fail --head http://localhost/images/ironic-python-agent.initramfs ; do sleep 1; done
-while ! curl --fail --head http://localhost/images/ironic-python-agent.kernel ; do sleep 1; done
+
+# Embed agent ignition into the rhcos live iso
+sudo podman run -d --net host --privileged --name configure-coreos \
+     --restart on-failure \
+     --env IRONIC_RAMDISK_SSH_KEY="$IRONIC_RAMDISK_SSH_KEY" \
+     --env PROVISIONING_INTERFACE=$PROVISIONING_NIC \
+     --entrypoint /bin/configure-coreos-ipa \
+     -v $AUTH_DIR:/auth:ro \
```

Comment:
Do we have TLS here? I think no, but if yes, we need to mount its volume here.

Reply:
I don't think we do TLS in the bootstrap node's ironic.